src/renderers/dom/client/eventPlugins/SimpleEventPlugin.js

```diff
@@ -137,6 +137,23 @@ function getDictionaryKey(inst) {
   return '.' + inst._rootNodeID;
 }
 
+function isInteractive(tag) {
+  return tag === 'button' || tag === 'input' ||
```

Comment:
Sorry, for whatever reason I totally caught your comment wrong before. Just a sec.